### PR TITLE
EOS-1060: Network switch monitoring using SNMPv3

### PR DESCRIPTION
**Pull request by sumedhak27**
-----
[View rendered Nw-Switch_Monotoring_using_SNMPv3/README.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/README.md)
[View rendered Nw-Switch_Monotoring_using_SNMPv3/docs/resources.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/docs/resources.md)
[View rendered Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_arch.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_arch.md)
[View rendered Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_get_impl.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_get_impl.md)
[View rendered Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_inform_impl.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/docs/snmp_inform_impl.md)
[View rendered Nw-Switch_Monotoring_using_SNMPv3/objective/objective.md](https://github.com/sumedhak27/cortx-experiments/blob/EOS-11060-Mellanox_SN2100_monitoring_using_SNMPv3/Nw-Switch_Monotoring_using_SNMPv3/objective/objective.md)

### CODE REVIEW

File: SNMPv3/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# SNMPv3 POC
```
Comment (cdeshmukh) [inline SNMPv3/README.md:1]: Rename POC, even as main topic in repo

**Network switch monitoring using SNMPv3**

File: SNMPv3/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Simple Network Management Protocol (SNMP) is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks and for monitoring or modifying that information to change device behavior.
+
+### Objective of the POC
+SSPL currently does not support the monitoring of private/public switches.
+This POC (Proof of Concept) displays how we can monitor switches (like
```
Comment (cdeshmukh) [inline SNMPv3/README.md:6]: This POC (Proof of Concept) demonstrate how we can monitor network switches (

File: SNMPv3/objective/objective.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+SSPL currently does not support the monitoring of private/public switches.
+This POC (Proof of Concept) displays how we can monitor switches (like
```
Comment (cdeshmukh) [inline SNMPv3/objective/objective.md:2]: same comment as earlier

File: SNMPv3/README.md (hunk 1 above)
Comment (cdeshmukh) [reply SNMPv3/README.md]: Folder name also needs to reflect this .. have folder name as NetworkSwitchMonitoring

**Review by cdeshmukh — COMMENTED** [review]
Content looks fine to me. Just parent folder name change to NwSwitchMonitoring-usings-SNMPv3 is recommended

File: SNMPv3/README.md (hunk 1 above)
Comment (sumedhak27) [reply SNMPv3/README.md]: done..

**Review by cdeshmukh — APPROVED** [review]

**Comment by sumedhak27** [comment]
@ajaysrivas  @sachinpunadikar please review.

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete

**Pull request closed without merging**